packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -758,7 +763,7 @@ const ruleset: any = {
     ProvisioningStateMustBeReadOnly: {
       description: "This is a rule introduced to validate if provisioningState property is set to readOnly or not.",
       message: "{{error}}",
-      severity: "off", // See https://github.com/Azure/azure-sdk-tools/issues/6071#issuecomment-1535560188
+      severity: "off", // See https://github.com/Azure/azure-sdk-tools/issues/6191#issuecomment-1571334585
```

Comment:
I realize this is a separate issue but should we use the strategy of demoting this to a staging only rule so we can still run it if needed for testing?

Reply:
I thought so too, but my read from the linked comment is there is no need to run this in staging: we know this is rule is borked and how to fix it. I guess once it is fixed, then we can re-enable it for staging.

Reply:
Done.